api/pyproject.toml

```diff
@@ -4,16 +4,16 @@ build-backend = "poetry.masonry.api"
 
 [tool.poetry]
 name = "doctr-api"
-version = "0.5.2a0"
+version = "1.0.1a0"
```

Comment:
:warning: This is the version of the package, not of poetry
I'd suggest reverting back

Reply:
Actually, I realize you want to bump docTr to 1.0.0. Perhaps that might be easier to discuss this in an issue and lower the amount of modifications in this specific PR?